site/docs/guide/files.md

```diff
@@ -181,12 +181,20 @@ new InputFile("/path/to/file");
 new InputFile(await Deno.open("/path/to/file"));
 ```
 
+```ts [Bun]
+// Send a local file.
+new InputFile("/path/to/file");
+
+// Send a `BunFile` instance.
+new InputFile(await Bun.file("/path/to/file").bytes());
```

Comment:
Why would
```suggestion
new InputFile(await Bun.file("/path/to/file"));
```
not work?

Reply:
Unfortunately no, we will have to fix this at the level of the grammY itself and add support for BunFile.
![image](https://github.com/user-attachments/assets/7b583384-2d7c-4629-bcf9-fff6255e282d)

Reply:
Then their docs are wrong: https://bun.sh/docs/api/file-io#reading-files-bun-file

They say that they conform to `Blob` so that means that `Bun.file("/tmp/file") instanceof Blob` should be `true`.

Reply:
Their reference is also wrong because their example states that `.bytes()` exists but their reference does not list it. Or is that implicit via `Blob` which they don't seem to inherit? Also, why did they list `.stream()` then?

Reply:
But well, I guess that you are right that we cannot perform this change